worker/src/Channel/ChannelRequest.cpp

```diff
@@ -197,4 +200,16 @@ namespace Channel
 
 		this->channel->Send(jsonResponse);
 	}
+
+	const std::string& ChannelRequest::GetNextInternalRoutingId()
```

Comment:
Perhaps "Pop" instead of "Get"? A method called "GetXxxx()" should not modify internal state and should be idempotent. "Pop" instead means remove and give me the last item.